src/Business Foundation/Test Library/NoSeries/src/LibraryNoSeries.Codeunit.al

```diff
@@ -11,7 +11,7 @@ codeunit 134510 "Library - No. Series"
         NoSeries.Code := NoSeriesCode;
         NoSeries.Description := NoSeriesCode;
         NoSeries."Default Nos." := true;
-        NoSeries.Insert();
+        if NoSeries.Insert() then;
```

Comment:
What about exiting this instead? if you expect a new no. series it's good to get an error, if you don't care whether it already exists then you can do if-then around this call.

Reply:
Same below